local-rendering.md

```diff
@@ -1 +1,36 @@
-# Serving the site locally
\ No newline at end of file
+# Serving the site locally
+
+## 1. Set the path of your content folder
+
+The first step to rendering your content folder with `NEBULA` is to set the `CONTENT_PATH` environment variable to the path of your content repository, *i.e.*,
```

Comment:
I would add a link to the "Creating a content repository" documentation here. So: 
```suggestion
The first step to rendering your content folder with `NEBULA` is to set the `CONTENT_PATH` environment variable to the path of your content repository (see the documentation on [setting up a content repository](https://github.com/esciencecenter-digital-skills/NEBULA-docs/blob/main/content-repo-instantiation.md)), *i.e.*,
```